src/test/ml-modules/root/test/suites/matching-json/suite-setup.xqy

```diff
@@ -22,3 +18,18 @@ return
     xdmp:default-permissions(),
     $const:CONTENT-COLL
   )
+;
+xquery version "1.0-ml";
+
+import module namespace const = "http://marklogic.com/smart-mastering/constants"
```

Comment:
Why two separate transactions for this? 

If there is a good reason, the "const" import can be dropped from this part, and the "matcher" import dropped from the first one.